contracts/bridge/BridgeTokens.sol

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.5.6;
+
+import "../externals/openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract BridgeTokens is Ownable {
+    uint constant private maxTokenCnt = 1000; // This count was derived by loop iteration gas consumption.
+    mapping(address => address) public allowedTokens; // <token, counterpart token>
+    address[] public allowedTokenList;
+    mapping(address => bool) public lockedTokens;
```

Comment:
Is using 2 mapping cheaper than using struct???
How about using struct like this?
struct token {
  tokenAddr
  counterPartAddr
  lock
}
mapping(address => token)

Reply:
It is good idea. But it can't return the list of the token. So I think this way is better than your suggestion.
What do you think? :)